src/Microsoft.AspNet.Authentication/DataHandler/TextEncoder.cs

```diff
@@ -2,18 +2,17 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.Authentication
```

Comment:
I want to hide this type *somewhere* - whether you want to mark it as `internal` or move it to a `.Internal` namespace, I'm fine either way.